src/components/POLine/Cost/CostForm.js

```diff
@@ -117,7 +124,16 @@ class CostForm extends Component {
           />
         </Col>
         <Col xs={6}>
-          <FieldCurrency />
+          <Field
+            component={Select}
+            fullWidth
+            label={<FormattedMessage id="ui-orders.cost.currency" />}
+            name="cost.currency"
+            placeholder=" "
+            dataOptions={currenciesOptions}
+            required
+            validate={[Required]}
```

Comment:
let's use lowercase version, `required`, since it's just a function, not a Class or Component

Reply:
we can't use required in lowercase as on 41 line there was defined variable with the same name